shared/base/view.js

```diff
@@ -398,6 +398,11 @@ module.exports = BaseView = Backbone.View.extend({
   },
 
   remove: function() {
+    // Remove reference to this view from its parentView
+    if (this.app.router.currentView != this) {
```

Comment:
since this will still error in the case that parentView is undefined - it might be better to just test that instead of if it's the `currentView` - so just having `if (this.parentView && this.parentView.childViews)` might be a better way to go.